Docs/SPE 260: update and reorganize telemetry metric references (#21965)
hashicorp/vault: website/content/partials/telemetry-metrics/vault/replication/rpc/standby/server/wrap_token_request.mdx

### vault.replication.rpc.standby.server.wrap_token_request ((#vault-replication-rpc-standby-server-wrap_token_request))

Metric type | Value | Description
----------- | ----- | -----------

> summary     | ms    | Time required to complete a wrap token request on the standby node

Reviewer: ```suggestion
summary     | ms    | Time required to service a wrap token request from a standby node
```

Can you apply this same pattern to the other rpc.standby.server metrics?

Author: fixed on all the standby.server metrics
